episodes/4-advanced-layer-types.Rmd

```diff
@@ -387,6 +396,8 @@ sns.lineplot(data=history_df[['loss', 'val_loss']])
 
 It seems that the model is overfitting somewhat, because the validation accuracy and loss stagnates.
 
+## 9. Tune hyperparameters
+
```

Comment:
For sake of completeness, should we add 10. save the model?

Reply:
Good point. Added!